test/lint/lint-shell.sh

```diff
@@ -46,15 +46,17 @@ if ! command -v yq > /dev/null; then
 fi
 
 EXCLUDE_GITIAN=${EXCLUDE}",$(IFS=','; echo "${disabled_gitian[*]}")"
+SHELLCHECK_CMD="shellcheck --external-sources --check-sourced $EXCLUDE_GITIAN"
 for descriptor in $(git ls-files -- 'contrib/gitian-descriptors/*.yml')
 do
-    echo
```

Comment:
I don't know is this is expected or changed here, but running this I see:
```bash
test/lint/lint-shell.sh
2020/04/25 19:36:23 unknown command "contrib/gitian-descriptors/gitian-linux.yml" for "yq"
2020/04/25 19:36:23 unknown command "contrib/gitian-descriptors/gitian-osx-signer.yml" for "yq"
2020/04/25 19:36:23 unknown command "contrib/gitian-descriptors/gitian-osx.yml" for "yq"
2020/04/25 19:36:23 unknown command "contrib/gitian-descriptors/gitian-win-signer.yml" for "yq"
2020/04/25 19:36:23 unknown command "contrib/gitian-descriptors/gitian-win.yml" for "yq"
```

Reply:
Could #17361 help? Did you have [`yq`](https://kislyuk.github.io/yq/) installed?

Reply:
> Did you have yq installed?

Yes. `yq version 3.3.0`

Reply:
> > Did you have yq installed?
> 
> Yes. `yq version 3.3.0`

Not [that](https://github.com/mikefarah/yq/releases) but [this](https://kislyuk.github.io/yq/) one:
```
$ brew install python-yq
```